tests/table/filter_evaluation_plugin.test.ts

```diff
@@ -352,6 +361,30 @@ describe("Filter criterion test", () => {
     return range(0, 10).filter((row) => model.getters.isRowFiltered(sheetId, row));
   }
 
+  test.each([
+    ["beginsWithText", ["a"], []],
+    ["endsWithText", ["a"], []],
+    ["isEqualText", ["a"], []],
+  ])(
+    "Can filter based on a text criterion %s ignoring lowercase/uppercase and whitespaces",
+    (type: string, criterionValues: string[], expectedFilteredRows: number[]) => {
+      const grid = {
+        A2: "a",
+        A3: " a",
+        A4: "a ",
+        A5: "A",
+      };
+      setGrid(model, grid);
+      createTableWithFilter(model, "A1:A5");
+      updateFilterCriterion(model, "A1", {
```

Comment:
this test passes even without this `updateFilterCriterion`, so I'm not sure what's the point